lib/et/api.rb

```diff
@@ -14,8 +14,16 @@ def initialize(options)
     end
 
     def list_lessons
-      response = Net::HTTP.get(lessons_url)
-      JSON.parse(response, symbolize_names: true)[:lessons]
+      request = Net::HTTP::Get.new(lessons_url)
```

Comment:
I'd try to break this up more - probably too little to be it's own class, but maybe a few private helpers?